pages/dkp/kaptain/2.0.0/install/air-gapped-dkp/index.md

```diff
@@ -82,13 +101,58 @@ Use the following instructions to install Spark Operator from Kommander Catalog
 
    Based on the network latency between the environment of script execution, the Docker registry, and the disk speed, this can take a while to upload all the images to your image registry.
 
-### Install Kaptain
-
-- When the Konvoy cluster is ready and the images are pushed to the registry, [install Kaptain][install-kaptain].
-
-[install-kaptain]: ../konvoy-dkp/
-[install-spark-dkp2]: /dkp/kommander/2.1/workspaces/applications/catalog-applications/dkp-applications/spark-operator/
-[install-spark-konvoy1]: /dkp/kommander/1.4/projects/platform-services/platform-services-catalog/kudo-spark/
-[kommander-install]: /dkp/kommander/2.1/install/air-gapped/
-[konvoy-air-gap]: /dkp/konvoy/2.1/choose-infrastructure/aws/air-gapped/
-[konvoy_deploy_addons]: /dkp/konvoy/1.8/upgrade/upgrade-kubernetes-addons/#prepare-for-addons-upgrade
+## Add Kaptain to your DKP Catalog Applications via CLI
```

Comment:
I feel like this should be in the DKP docs instead/as well.

Reply:
@cmastr 

Should we add a mini-topic under **Kommander > Workspaces > Applications > Catalog Apps > DKP Apps** called `Install Kaptain` that links to this section here? And under **Kommander > Projects > Applications > Catalog Apps > DKP Apps**?

Reply:
@cmastr 

Should we add a mini-topic under **Kommander > Workspaces > Applications > Catalog Apps > DKP Apps** called `Install Kaptain` that links to this section here? And under **Kommander > Projects > Applications > Catalog Apps > DKP Apps**?